src/vaadin-combo-box-data-provider-mixin.html

```diff
@@ -200,7 +200,7 @@
     }
 
     _pageSizeChanged(pageSize, oldPageSize) {
-      if (Math.floor(pageSize) !== pageSize || pageSize === 0) {
+      if (Math.floor(pageSize) !== pageSize || pageSize === 0 || pageSize < 0) {
```

Comment:
Isn't `pageSize === 0 || pageSize < 0` the same as `pageSize < 1` which makes sense (pagesize shouldn't be less than 1) and reads better

Reply:
Valid point 😄 I will update accordingly, thanks for the feedback @tomivirkki 🙏